.github/workflows/build.yml

```diff
@@ -15,6 +15,8 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: '0'
```

Comment:
iirc by default we do a super-shallow clone with no history at all, but if you do fetch-depth 0 honestly i forget what it means, but it means "fetch me enough git history that i can find the previous commit referenced in the `  new-from-rev` tag"